### PR TITLE
Add fallback text for store description

### LINKED ISSUE #390: No addional information in Akzeptanzstellen list
**Describe the bug**

**To Reproduce**
1. Go to 'Suche'
2. See the Suchresultate

**Expected behavior**
The description should be shown in each line

**Screenshots**
image

**Smartphone (please complete the following information):**
 - Device: iPhone 5s (iphone a1457)
 - iOS 12....

### PR DESCRIPTION
**Pull request by klinzo**
Fixes #390 

Apparently it was just bad luck that all three search results had no description in the screenshot in the linked issue, the description should always be shown if it is available (also on small devices).
Since it looked kind of weird though, I added the text "Keine Beschreibung verfügbar" as fallback description for store preview and search results.
I did NOT add the fallback text to the store detail widget.

<div>

</div>

I also fixed some typos, outdated docu and links.

### CODE REVIEW

File: frontend/.idea/frontend.iml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module type="WEB_MODULE" version="4">
+<module type="JAVA_MODULE" version="4">
```
Comment (klinzo) [inline frontend/.idea/frontend.iml:2]: Not sure about this change, was automatically done by IntelliJ

File: frontend/lib/map/preview/accepting_store_preview.dart [hunk 2]
```diff
@@ -52,7 +52,7 @@ class AcceptingStorePreview extends StatelessWidget {
     return AcceptingStoreSummaryModel(
         item.id,
         item.store.name,
-        item.store.description ?? "Keine Beschreibung verfübar",
```
Comment (Schedulaar) [inline frontend/lib/map/preview/accepting_store_preview.dart]: Why did you remove this again? (Also there was a typo in "verfübar")

**Review by Schedulaar — APPROVED** [review]

File: frontend/lib/map/preview/accepting_store_preview.dart (hunk 2 above)
Comment (klinzo) [reply frontend/lib/map/preview/accepting_store_preview.dart]: Was unnecessary since the frontend/lib/store_widgets/accepting_store_summary.dart widget is used in both places.

Comment (Schedulaar) [reply frontend/lib/map/preview/accepting_store_preview.dart]: Ah, I see 👍🏽

File: frontend/.idea/frontend.iml (hunk 1 above)
Comment (maxammann) [reply frontend/.idea/frontend.iml:2]: It should stay a WEB_MODULE

File: CODEOWNERS [hunk 3]
```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for
 # review when someone opens a pull request.
-*       @schedulaar @RoAnBu @elKei24 @maxammann @aiquita @PaulaWi
+*       @schedulaar @RoAnBu @elKei24 @maxammann @aiquita @PaulaWi @klinzo
```
Comment (maxammann) [inline CODEOWNERS:7]: I think you can remove all except for @schedulaar, @klinzo and @maxammann

At least @aiquita wants to be removed afaik

File: frontend/.idea/frontend.iml (hunk 1 above)
Comment (klinzo) [reply frontend/.idea/frontend.iml:2]: Will revert

Comment (maxammann) [reply frontend/.idea/frontend.iml:2]: If intellij keeps changing this then something is weird. Which version are you using?

File: frontend/.idea/runConfigurations.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RunConfigurationProducerService">
+    <option name="ignoredProducers">
+      <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
+      </set>
```
Comment (maxammann) [inline frontend/.idea/runConfigurations.xml:7]: Should be removed

File: frontend/lib/store_widgets/accepting_store_summary.dart [hunk 5]
```diff
@@ -164,7 +164,7 @@ class StoreTextOverview extends StatelessWidget {
                 overflow: TextOverflow.ellipsis,
                 style: Theme.of(context).textTheme.bodyText1),
             const SizedBox(height: 4),
-            Text(store.description ?? "",
+            Text(store.description ?? "Keine Beschreibung verfügbar",
```
Comment (maxammann) [inline frontend/lib/store_widgets/accepting_store_summary.dart:167]: Maybe just "Keine Beschreibung" to keep it even shorter?

**Review by maxammann — APPROVED** [review]

File: frontend/.idea/frontend.iml (hunk 1 above)
Comment (klinzo) [reply frontend/.idea/frontend.iml:2]: IntelliJ? The latest version, 2021.2.3. The `runConfiguration` file is also added all the time...

File: frontend/lib/store_widgets/accepting_store_summary.dart (hunk 5 above)
Comment (klinzo) [reply frontend/lib/store_widgets/accepting_store_summary.dart:167]: Imo `Keine Beschreibung verfügbar` is better, but on smaller devices it is cut anyway with `...`. I don't really care though, not sure who has expertise here.

Comment (maxammann) [reply frontend/lib/store_widgets/accepting_store_summary.dart:167]: I'm fine with both so lets leave it :)
